docs/ui/components/ExpoStyling/CompatibilityTable.tsx

```diff
@@ -0,0 +1,216 @@
+import { css } from '@emotion/react';
+import { matchSorter } from 'match-sorter';
+import { useMemo } from 'react';
+import { useTable, useFilters, useGroupBy, useExpanded } from 'react-table';
```

Comment:
What's the reason for adding an additional table component? We have our own component already, which is also hooked us to Remark MD(X) parser, ale we plan to replace in the future with new one, which is already developed for dashboard.

Reply:
This is a 1000+ line table with expandable rows. Because its so large I've also added grouping and filtering per column. Would the existing table handle this functionality? I wasn't able to locate a component that did.  As `react-table` is a headless library, it is being styled using the `<Table />` components from `'~/ui/components/Table'`

Reply:
Hmm, so in this case we probably need to ask different question - what we can do to do not display 1000 rows+ table on one page? How can we divide it and, in general, make easier to consume information contained in there for docs users? Or maybe we can use external resource to redirect people to? 

Saying that, I'm not familiar with the content of this table, need to look at this locally, was making only a quick check via GH diffs.

Reply:
The content is the same as http://nativewind.dev documentation. Its a table comparing what is available across native and web, pointing to the external TailwindCSS docs for more information. There are ~1100 classes that we need to document.

For reference, http://nativewind.dev broke up this comparison across multiple page - mimicing the Tailwind CSS doc's layout. The feedback that I received was that these pages are inefficient and difficult to gauge what is/isn't supported.

Maybe we should do a huddle to talk able this indepth?

Reply:
I was just going through the table and it's really big. 

- I don't like the design changes it introduces (it doesn't resonate with the existing overall design we've for docs) and the empty blocks (for example in the first and the second column)

<img width="912" alt="CleanShot 2023-06-02 at 01 57 27@2x" src="https://github.com/expo/expo/assets/10234615/c6652790-6fe6-4870-a499-e81d27f8ae8d">

- The notes toggling shifts the layout

 
<img width="882" alt="CleanShot 2023-06-02 at 01 58 00@2x" src="https://github.com/expo/expo/assets/10234615/c60f3bb4-fcd9-4b94-9abf-15ddd0e1573c">

- Another option is to use existing components. That way we can leverage the TOC in the side bar for people to quickly navigate between different sections:

![CleanShot 2023-06-02 at 01 59 20@2x](https://github.com/expo/expo/assets/10234615/b35b8f57-d733-4c88-974f-19e90633a065)